### PR TITLE
38 qu topoinputsinp

### PR DESCRIPTION
**Pull request by ntutland**

### CODE REVIEW

File: quicfire_tools/topography.py
Comment (amarcozzi) [inline quicfire_tools/topography.py:1]: I think we need to add a __str__ method for each TopoType class in order to write the topography inputs to the QU_TopoInputs.inp file.

For example if you run the following:
```python
topoinputs = self.get_default_test_object()
gaussian_hill = GaussianHillTopo(
    x_hilltop = 100, y_hilltop = 150, elevation_max = 500, elevation_std = 20)
topoinputs.topo_type = gaussian_hill
topoinputs.to_file("tmp/")
```

Then you get the following output file:

```
Specify file name for custom topo (full path)
topo.dat
1		! N/A, topo flag: 0 = flat, 1 = Gaussian hill, 2 = hill pass, 3 = slope mesa, 4 = canyon, 5 = custom, 6 = half circle, 7 = sinusoid, 8 = cos hill, 9 = QP_elevation.inp, 10 = terrainOutput.txt (ARA), 11 = terrain.dat (firetec)
2   ! N/A, smoothing method (0 = none (default for idealized terrain), 1 = blur, 2 = David's)
500   ! N/A, number of smoothing passes
200     ! N/A, number of initial SOR iterations (only if fire is run)
4         ! N/A, sor cycles
0.9          ! N/A, sor relaxation parameter (default for flat is 1.78, ignored if there is no terrain)
0                   ! N/A, add slope flow correction (0 = no, 1 = yes)
Command to run slope flow code (system specific)
""
```

Where the flag is correct but it is missing the Gaussian Hill inputs.

File: tests/test_input_module.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 Test module for the inputs module of the quicfire_tools package.
 """
-
 from pathlib import Path
 
 import pytest
 from pydantic import ValidationError
 
+import sys
+sys.path.insert(0, "/Users/ntutland/Documents/Projects/quicfire-tools")
```
Comment (amarcozzi) [inline tests/test_input_module.py:10]: Are you still having issues importing quicfire-tools into tests?

**Review by amarcozzi — CHANGES_REQUESTED** [review]

File: tests/test_input_module.py (hunk 1 above)
Comment (ntutland) [reply tests/test_input_module.py]: Nope  just forgot to take that out

**Review by amarcozzi — APPROVED** [review]
Looks good!

**Comment by amarcozzi** [comment]
Fix #38

**Pull request merged**